tests/test_views.py

```diff
@@ -24,376 +24,325 @@
 
 from reana_server.utils import _create_and_associate_reana_user
 
+GET_USER_MOCK = Mock(return_value=Mock(is_authenticated=False))
```

Comment:
We could extract this and the mock:
```python
with patch("flask_login.utils._get_user", GET_USER_MOCK):
    ...
```
To a new fixture in `conftest` so the test runs inside this context, for example [here](https://github.com/reanahub/pytest-reana/blob/master/pytest_reana/fixtures.py#L319) we do it. I think it will simplify a bit more the changes.